modules/apps/collaboration/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/select_file_entry.jsp

```diff
@@ -102,11 +102,11 @@ dlSearchContainer.setResults(foldersAndFileEntriesAndFileShortcuts);
 								Map<String, Object> data = new HashMap<String, Object>();
 
 								data.put("entryid", fileEntry.getFileEntryId());
-								data.put("entryname", fileEntry.getTitle());
+								data.put("entryname", HtmlUtil.unescape(fileEntry.getTitle()));
 								%>
 
 								<aui:a cssClass="selector-button" data="<%= data %>" href="javascript:;">
-									<%= HtmlUtil.escape(fileEntry.getTitle()) %>
+									<%= fileEntry.getTitle() %>
```

Comment:
Shouldn't we do the same in line https://github.com/sergiogonzalez/liferay-portal/pull/4251/commits/b168c253b5b6e5343e4bbf69ce7dca83e8b19566#diff-5015908b202ce75a3ab03a645dc27c36R114 ?

Reply:
Hello @sergiogonzalez ,

That is an oversight that I failed to catch; I have included the change. I will resend the PR with this change included.